itsy.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
+header {
+  display: flex;
+  background: url(img/lead.jpg);
```

Comment:
not sure that background image should be in header tag, it doesn't related to that.